scripts/DMF/xmlify-mib.py

```diff
@@ -107,16 +107,22 @@ def mk_snmp (inp, root):
                 kwargs ["multiplier"] = info ["info_len"]
 
             # I detect some diferences with the original structures if info_flags is ignored!
-            for info_flag in (ST_FLAG_STRING, ST_FLAG_RW, 0):
-                if info_flag in info ["info_flags"]:
-                    info ["info_flags"].remove (info_flag)
-
+            for name, info_flag, value in (
+                    ("writable", ST_FLAG_RW, "yes"),
+                    ("string", ST_FLAG_STRING, "yes"),
+                    ):
+                if not info_flag in info ["info_flags"]:
+                    continue
+                kwargs [name] = value
+                info ["info_flags"].remove (info_flag)
+            
             # This is a assert - if there are info_flags we do not cover, fail here!!!
-            if len (info ["info_flags"]) > 0:
-                die ("There are unprocessed items in info_flags in '%s'" % (info, ))
+            #if len (info ["info_flags"]) > 0:
```

Comment:
I agree, the die should not be removed. If anything was changed, script won't hit this and if yes, it will report it.